pkg/loader/load_test.go

```diff
@@ -413,13 +413,13 @@ func (s *txnManagerSuite) TestRunTxnManager(c *check.C) {
 	case t := <-output:
 		txnManager.pop(t)
 		c.Assert(t, check.DeepEquals, txn)
-	case <-time.After(50 * time.Microsecond):
+	default:
 		c.Fatal("Fail to pick txn from txnManager")
 	}
```

Comment:
```go
select {
case t := <-output:
case <-time.After...
}
```

This pattern appears a lot in this test, is it possible to extract this as a helper function to simplify the test?